src/components/bottom-sheet/BottomSheetContext.tsx

```diff
@@ -69,18 +68,14 @@ export const BottomSheetProvider: React.FC = ({children}) => {
   const close = () => {
     setContentFunction(() => () => null);
     setIsOpen(false);
-    if (closeRef) {
-      setTimeout(() => giveFocus(closeRef), 200);
-    }
+    onCloseFocusRef && giveFocus(onCloseFocusRef);
```

Comment:
No longer necessary to wrap `giveFocus` with `setTimeout`?

Reply:
No longer necessary it seems